### DISCUSSION TITLE
Idea > Usermanager

### DISCUSSION THREAD

**Discussion by Hutch79** [JackBailey/Lynx#65] [Ideas]
A way to manage users inside the web panel.

## Why?
It is already possible to enable/disable the registration of new users, but managing them is (at least in the web panel) currently not possible.

## Features
Even if the registration is disabled, it should be possible to register new users.
Deleting users should also be possible.

Besides of that (and analytics) this project is really nice and easy to use as well as set up!
I previously used Yourls which already has basic statistics, but Lynx feels much snappier/faster and also looks way better! So I definitely don't regret the change.

**Comment by JackBailey** [comment]
Glad you're enjoying Lynx. [User Management](https://github.com/Lynx-Shortener/Lynx/tree/user-management) is currently being worked on, I'm hoping to get some time to complete it within the next few weeks.

**Reply by Hutch79** [reply]
Add a very fast responding dev to the positive things as well ^^

Nice to hear that!
I think this will add a lot to this project, for example if Lynx is used in a company!

**Comment by JackBailey** [comment]
This has been added in [1.7.0](https://github.com/Lynx-Shortener/Lynx/pull/76)
